tox.ini

```diff
@@ -8,7 +8,9 @@
 ; * withcov - run the tests wrapped using the coverage.
 ; * nocov - run the tests directly, without using the coverage wrapper.
 ; * wheel - build the wheel distribution
-; * posix - prepare the tests to be run in a Linux/Unix/OSX environment.
+; * posix - prepare the tests to be run in a Linux/Unix/macOS environment.
```

Comment:
okay this explosion of environments is getting kind of ridicuous.  If we can't use platform specifiers, I'm starting to think we should include a `Makefile` or _something_ that does the dance you need to figure out what tox environments are appropriate to run locally.  I have a shell script that does this right now:

```bash
#!/bin/bash
cd ~/Projects/Twisted;
coverage erase;
detox $(tox -l | grep withcov-posix |
            while read x;
            do
                echo " -e ${x}";
            done
    ) -e apidocs \
      -e narrativedocs \
      -e topfile \
      -e manifest-checker \
      -e txchecker-travis;
coverage combine;
coverage html;
```

This is not a blocker for this branch but it's something we should do soon.